roaring/roaring.go

```diff
@@ -101,6 +102,14 @@ func (b *Bitmap) Add(a ...uint64) (changed bool, err error) {
 
 	return changed, nil
 }
+func (b *Bitmap) String() string {
```

Comment:
I am not sure what the go best practice is in regards to 1.  I personally like to have all the public/exported functions grouped together followed by the private functions.  You have anything other than "wierd"?